src/uikit/SummaryTable/VariableSummaryTable/Tables.js

```diff
@@ -34,9 +34,32 @@ const SummaryContent = styled('div')`
   padding-left: 10%;
 `;
 
-const Tables = ({ rows, amount }) => {
-  const numOfTables = amount === 1 ? rows.length : rows.length / amount;
-  const tables = chunk(rows, numOfTables);
+/**
+ * Splits a number into smaller chunks that will be even-ish. Returns the largest chunk, allowing lodash.chuck to do the
+ * heavy lifting from then on.
+ *
+ * Example: 11/3: 4/4/3 -> 4. 12/3: 4/4/4 -> 4.
+ *
+ * @param number The number to split
+ * @param amount The divider (the number of chunks we want)
+ * @returns {number} The biggest chunk.
+ */
+function splitter(number, amount) {
+  const approx = number / amount;
+
+  if(Number.isInteger(approx)) return approx;
+
+  const integer = Math.floor(approx);
+  const decimal = "0."+(approx+'').split('.')[1]; //weird, but best way to get precise decimal places https://stackoverflow.com/questions/4512306/get-decimal-portion-of-a-number-with-javascript
```

Comment:
Sad but true. Damn those JS unreliable floats. `const decimal = approx % 1;` would be so elegant!